app-editors/joe/joe-4.4.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Terminal-based text editor"
+DESCRIPTION="Joe's Own Editor is a full-featured text editor with minimal requirements. \
+It is fully customizable (Emacs, pico or WordStar are preset). Is also has multiple \
+windows, hexadecimal edit mode for binary files or syntax highlighting for more than 40 languages. \
+It also supports mouse for both text and window manipulation."
+HOMEPAGE="http://joe-editor.sourceforge.net/"
+LICENSE="GNU GPL v1"
+COPYRIGHT="1992, 2004, 2006 Joseph H. Allen"
```

Comment:
I just copied/pasted these from previous joe recipe, which was in the old bep format still.  I will adjust before final commit.